apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/Body.tsx

```diff
@@ -163,24 +170,30 @@ const Body = ({ data, onClose }: { data: Data; onClose?: () => void | undefined
           return onCancel(new DisabledTransactionBroadcastError());
         }
         onResult(operation);
-        // else not swap i.e card and sell we close the drawer
+      } else if (
+        (data.exchangeType === ExchangeType.SELL || data.exchangeType === ExchangeType.SELL_NG) &&
+        sourceCurrency
+      ) {
+        onResult(operation);
```

Comment:
We need to set DISABLE_TRANSACTION_BROADCAST logic too to avoid to have false errors. (i think we need to use the same flow than swap use: 
setResult
if DISABLE_TRANSACTION_BROADCAST
onResult

```
if (getEnv("DISABLE_TRANSACTION_BROADCAST") 
```

Reply:
It is not that important, the flow can go on in this case, but i will add for security.